packages/web/src/services/env/env.dev.ts

```diff
@@ -41,6 +41,7 @@ export const env: Env = {
   IDENTITY_SERVICE: 'http://audius-protocol-identity-service-1',
   INSTAGRAM_APP_ID: '2875320099414320',
   INSTAGRAM_REDIRECT_URL: 'http://localhost:3000',
+  LOOKUP_TABLE_ADDRESS: 'A32QiWMjwpeBpYYfr2MytJSuiTLE37VAwMWH4jG12YBd',
```

Comment:
interesting - do we want to add the lookup table to the Solana validator setup script for local dev? I think we should probably leave it out for local no?

Reply:
Oh right the dev env file is just for local validator isn't it? We use stage for actual local dev?

Reply:
not sure what "actual local dev" is 😅 but I generally test client changes against stage yeah?

Reply:
Set dev env values to empty string. I realize now that the table won't exist on the validator, so yeah that's not a good idea :-)

Reply:
does adding an empty string address for a lookup table work? I figured it wouldn't but could be wrong?

Reply:
I think it needs to be nullable or something? Or it could be an array of addresses and an empty one?

Reply:
Empty string will be fine. We check for truthy values where we use it. And we are adding it to an existing array when it is used.

Reply:
nice, i always forget empty string is falsey